src/arrayReverse.test.js

```diff
@@ -1,20 +1,38 @@
-'use strict';
+"use strict";
 
 describe(`Function 'arrayReverse':`, () => {
-  const arrayReverse = require('./arrayReverse');
+  const arrayReverse = require("./arrayReverse");
 
   it(`should be declared`, () => {
     expect(arrayReverse).toBeInstanceOf(Function);
   });
 
   it(`should return an array`, () => {
-
+    expect(arrayReverse([])).toBeInstanceOf(Array);
   });
 
   it(`should return an empty string
     if original array consists of an empty string`, () => {
```

Comment:
The description of the test case mentions 'an empty string', but the test checks for an array of spaces. Consider clarifying the test description to match the test case.